### DISCUSSION TITLE
The client application name is not configurable.

### DISCUSSION THREAD

**Discussion by SURAJTHEGREAT** [FreeOpcUa/python-opcua#1214] [Ideas]
Currently the client application name is hardcoded as Pure Python Client  as below.

https://github.com/FreeOpcUa/python-opcua/blob/master/opcua/client/client.py#L122 

Can this be made configurable.

Can we add a new argument name in the init and use this argument for a configurable name.

**Comment by AndreasHeine** [comment]
```python
client = Client()
client.name = "yourname"
```

**Reply by SURAJTHEGREAT** [reply]
thanks..that should help
